src/types/plugin-inputs.ts

```diff
@@ -18,7 +18,13 @@ export interface PluginInputs<T extends SupportedEventsU = SupportedEventsU, TU
  * The kernel will extract those and pass them to the plugin,
  * which are built into the context object from setup().
  */
-export const pluginSettingsSchema = T.Object({});
+
+export const pluginSettingsSchema = T.Object({
+  model: T.String({ default: "o1-mini" }),
+  openAiBaseUrl: T.Optional(T.String()),
+  similarityThreshold: T.Number({ default: 0.1 }),
```

Comment:
Can you explain to me what the similarity threshold is for?

Reply:
This could benefit from clarifying that this is calculating the difference with subtraction, so closer to 0 difference means more similar. Or to make it more intuitive, maybe reverse it. 

1 is the most similar and 0 is least similar, so you want 90% similarity threshold (0.9) thats a lot more intuitive for a config.